src/coreclr/jit/importer.cpp

```diff
@@ -14084,9 +14187,38 @@ void Compiler::impImportBlockCode(BasicBlock* block)
                     uns = false;
                 }
 
-                // At this point uns, ovf, callNode are all set.
+// At this point uns, ovf, callNode are all set.
```

Comment:
Please remove this. There should be no architecture-specific code in the importer.

Reply:
had removed.
Thanks